account_financial_report_qweb/report/general_ledger.py

```diff
@@ -325,11 +322,11 @@ def _get_account_sub_subquery_sum_amounts(
             """
         sub_subquery_sum_amounts += """
         LEFT JOIN
-            res_currency c ON ml.currency_id = c.id        
+            res_currency c ON a.currency_id = c.id
```

Comment:
Here we still have the comments in https://github.com/SimoRubi/account-financial-reporting/pull/1#discussion_r171212707. 
GitHub collapsed those comments .

Reply:
Yes, but do you agree with my responses there?